integreat_cms/cms/templates/events/event_list_row.html

```diff
@@ -116,13 +116,13 @@
             <i icon-name="calendar"></i> {% translate "One-time" %}
         {% endif %}
     </td>
-    {% if perms.cms.view_externalcalendar %}
-        <td class="py-3 pr-2">
-            {% if event.external_calendar %}
-                {{ event.external_calendar.name }}
-            {% endif %}
-        </td>
-    {% endif %}
+    <td class="py-3 pr-2">
+        {% if event.external_calendar %}
+            {{ event.external_calendar.name }}
+        {% else %}
+            <i icon-name="minus"></i>
+        {% endif %}
```

Comment:
Is there a deliberate reasoning for showing a dash instead of just leaving it blank? Should we maybe adapt the same for empty fields in the new contacts list view?

Reply:
It was just a spontaneous idea from me side and hasn't been discussed before. I'm willing to change it back, if it's preferred. It just had a personal preference for a dash instead of an empty field

Reply:
So, it was agreed that this is better than leaving it blank. We still have a discussion whether or not it's better to center it or not.
As for the other list views: In feedback we already have it, in fact, that's where I go the idea from :)

Reply:
It was agreed that left alignment is better than center :)